src/Administration/Resources/app/administration/src/module/sw-customer/page/sw-customer-list/sw-customer-list.scss

```diff
@@ -10,7 +10,7 @@ $sw-customer-list-color-success: $color-emerald-500;
         position: absolute;
     }
 
-    .sw-data-grid__cell:not(.sw-data-grid__cell--actions) {
+    .sw-data-grid__cell:not(.sw-data-grid__cell--actions):not(.sw-data-grid__cell--header) {
```

Comment:
This and all other selectors can be shortened to `.sw-data-grid__cell:not(.sw-data-grid__cell--header)`. As the action cell is also a header cell.

Reply:
Will amend that, ty :)